magento/magento2
# [Magento 2.2] Upload fails in image uploader

### Description
The issue can be reproduced only in case if `fileinfo` module is disabled or not installed. Problem is that mime type of uploaded file would be resolved by `tmp` file. And in case of disabled `fileinfo` module `Uploader` will try to resolve mime type by file extension that always equals null for `tmp` files

### Fixed Issues

1. magento/magento2#16531: [Magento 2.2.5] Upload fails in image uploader

### Manual testing scenarios (*)

1. Ensure that `fileinfo` module is disabled  
2. Open Cms Block edit page
3. Select the image uploader ("Insert Image" button or the one in the editor)
4. Select an image from your hd and upload it.

#### Expected result
An added image in the gallery

#### Actual result
An error is shown: "File validation failed." No image is uploaded.

### Contribution checklist (*)
